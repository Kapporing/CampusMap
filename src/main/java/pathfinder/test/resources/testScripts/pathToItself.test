# Create a single node and test a path to itself
CreateGraph graph

AddNode graph node1

AddEdge graph node1 node1 0

FindPath graph node1 node1
