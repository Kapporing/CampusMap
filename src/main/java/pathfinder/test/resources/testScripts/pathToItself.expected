# Create a single node and test a path to itself
created graph graph

added node node1 to graph

added edge 0.000 from node1 to node1 in graph

path from node1 to node1:
total cost: 0.000
